common/src/main/java/io/novafoundation/nova/common/utils/ViewSpace.kt

```diff
@@ -0,0 +1,31 @@
+package io.novafoundation.nova.common.utils
+
+import android.view.View
+import android.view.ViewGroup
+
```

Comment:
Sounds very similar to Rect or RectF

Reply:
Yes but it's from android graphic library. I don't want to mix it